rclcpp/test/rclcpp/test_executor.cpp

```diff
@@ -508,3 +509,25 @@ TEST_F(TestExecutor, is_spinning) {
 
   ASSERT_TRUE(timer_called);
 }
+
+TEST_F(TestExecutor, release_ownership_entity_after_spinning_cancel) {
```

Comment:
IMO this test should be run for all executors (using the parametrized tests in `test_executors.cpp`

Reply:
Yes. I will updated test codes.